src/set_theory/game/pgame.lean

```diff
@@ -607,34 +607,42 @@ begin
   congr; funext i; cases i
 end
 
-/-- An explicit equivalence between the moves for Left in `-x` and the moves for Right in `x`. -/
--- This equivalence is useful to avoid having to use `cases` unnecessarily.
-def left_moves_neg (x : pgame) : (-x).left_moves ≃ x.right_moves :=
+@[simp] theorem left_moves_neg (x : pgame) : (-x).left_moves = x.right_moves :=
```

Comment:
Is this actually useful as a `@[simp]` lemma? I'm unconvinced that `simp` would ever be able to use these without running into dependent type theory issues. (They are not currently used by `simp` anywhere.)

I think the fact these were equivalences not equalities was intentional: equations between types aren't actually that useful if they are not definitional, and requiring the user to work through the equivalence API helps more than it hurts.

Reply:
You can certainly cook up use cases (suppose you wanted to prove `is_empty ((-x).left_moves)`, for instance), so I don't think having these as `simp` is useless. I agree that the equivalences are more useful, which is why I kept them in, but I still think this theorem is nice to have.

Reply:
On a related note, I'd like to apologize for the Frankenstein multipurpose PR. The original goal was just to switch the order of the equivalence to make the basic API a bit friendlier. This meant I had to tweak a lot of proofs, and I ended up finding a lot of golf opportunities.

Reply:
Then maybe `local attribute [simp] your_lemmas` where you use them?

Reply:
Is there anything wrong with just marking them as `simp`?

Reply:
Bump. While I understand why you struck down other of my type equality simp lemmas, I'm not really seeing it on this one. The latter is just a strictly simpler type to work with. 

That said, if it will move this PR along, I'll remove the `simp`.

Reply:
This is because equality of types, unless defeq, will only cause trouble. It will introduce a bunch of ineffable `eq.rec` in the goal. It's actually very rare to have two types propositionally but not definitionally equal. You found one such case.

Reply:
`simp` has been removed.